plugins/VisitFrequency/lang/en.json

```
@@ -1,39 +1,39 @@
+        "ColumnActionsByNewVisits": "Actions by new visitors",
+        "ColumnActionsByReturningVisits": "Actions by returning visitors",
+        "ColumnAverageVisitDurationForNewVisitors": "Avg. amount of seconds spent by new visitors",
+        "ColumnAverageVisitDurationForReturningVisitors": "Avg. amount of seconds spent by returning visitors",
+        "ColumnAvgActionsPerNewVisit": "Avg. actions per new visitor",
+        "ColumnAvgActionsPerReturningVisit": "Avg. actions per returning visitor",
+        "ColumnBounceCountForReturningVisits": "Number of bouncing returning visitors",
+        "ColumnBounceRateForNewVisits": "Bounce rate for new visitors",
+        "ColumnBounceRateForReturningVisits": "Bounce rate for returning visitors",
+        "ColumnMaxActionsInReturningVisit": "Max actions in one returning visit",
+        "ColumnNbReturningVisitsConverted": "Number of converted returning visitors",
+        "ColumnNewVisits": "New visitors",
+        "ColumnReturningVisits": "Returning visitors",
+        "ColumnSumVisitLengthReturning": "Total amount of seconds spent by returning visitors",
         "ColumnUniqueNewVisitors": "Unique new visitors",
         "ColumnUniqueReturningVisitors": "Unique returning visitors",
-        "ColumnNewUsers": "New Users",
-        "ColumnReturningUsers": "Returning Users",
-        "NewActions": "actions by the new visits",
-        "NewAverageVisitDuration": "average visit duration for new visitors",
-        "NewAvgActions": "actions per new visit",
-        "NewBounceRate": "new visits have bounced (left the website after one page)",
+        "ColumnNewUsers": "New users",
+        "ColumnReturningUsers": "Returning users",
+        "NewActions": "actions by the new visitors",
+        "NewAverageVisitDuration": "avg. time spent by new visitors",
+        "NewAvgActions": "actions per new visitor",
+        "NewBounceRate": "new visitors have bounced (left the website after one page)",
         "NewVisits": "new visits",
-        "PluginDescription": "Reports metrics about your first time new visitors and returning visitors.",
-        "ReturnActions": "actions by the returning visits",
-        "ReturnAverageVisitDuration": "average visit duration for returning visitors",
-        "ReturnAvgActions": "actions per returning visit",
-        "ReturnBounceRate": "returning visits have bounced (left the website after one page)",
-        "ReturningVisitDocumentation": "A returning visit is (as opposed to a new visit) made by someone who has visited the website at least once before.",
-        "ReturningVisitsDocumentation": "This is an overview of the returning visits.",
-        "ReturnVisits": "returning visits",
+        "PluginDescription": "Reports metrics about your returning and (first time) new visitors.",
+        "ReturnActions": "actions by the returning visitors",
+        "ReturnAverageVisitDuration": "avg. time spent by returning visitors",
```

Also counting prior visits
```suggestion
        "ReturnAverageVisitDuration": "avg. time spent by returning visitors in last visit",
```
?

---

It counts all visits and one visitor can have had multiple visits. The avg visit duration is calculated looking at each visit in separate. So if a visitor visits the page e.g. four times for a minute, the avg visit duration would still be a minute, while avg time spent sounds like it should be four minutes maybe.